src/main.ts

```diff
@@ -98,8 +112,12 @@ async function run(): Promise<void> {
     if (shouldSkipBranchLint(headBranch, BRANCH_IGNORE_PATTERN)) {
       process.exit(0);
     }
+    const shouldUseCustomRegexp = !!CUSTOM_ISSUE_NUMBER_REGEXP && !!JIRA_PROJECT_KEY;
```

Comment:
so is it like - to use custom regex I need both `custom-issue-number-regexp` and `jira-project-key`?

Can we add a check for that?
If one is present and the other is not, then throw a warning